src/app/app.component.ts

```diff
@@ -1,10 +1,112 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Item } from './models/item';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'angular-green-grocers';
-}
+  groceries: Item[] = [
```

Comment:
We shouldn't be hardcoding the data here - this wouldn't be possible to do in a real-world application